src/controllers/timeSheet.js

```diff
@@ -98,10 +98,14 @@ const getAllTimesheets = async (req, res) => {
     }
     if (req.query.hours) {
       filterByParams = filterByParams.filter(
-        (timesheet) => timesheet.hours === parseInt(req.query.hours, 10)
+        (timesheet) => timesheet.hours === parseInt(req.query.hours, 10),
       );
     }
-    return res.status(200).json({ filterByParams });
+    return res.status(200).json({
```

Comment:
It still returns an empty array if a nonexistent attribute is entered in the filters. This fixes it (after line 103 and before the "catch")
 
```suggestion
        if (filterByParams.length !== 0) {
      return res.status(200).json({
        message: 'Timesheet found',
        data: filterByParams,
        error: false,
      });
    }
    return res.status(404).json({
      message: 'Timesheet not found',
      error: true,
    });
```

Reply:
Fixed! I find other way to do it without change the tests